.github/workflows/scripts/test/test_translation_check.py

```
@@ -0,0 +1,187 @@
+    def test_find_tags_dynamic_ignored(self):
+        """Verify dynamic variables are ignored."""
+        self.assertEqual(
+            len(translation_check.find_translation_tags("t(var)")), 0
+        )
+
+    def test_get_files_directory(self):
+        """Verify directory scanning."""
+        src = Path(self.test_dir) / "src"
+        src.mkdir()
+        (src / "app.tsx").touch()
+        self.assertEqual(
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)